templates/autoscale-tgw-new-vpc.template.yaml

```diff
@@ -7,362 +7,119 @@ Description: >-
     FortiGate On-Demand instances and related AWS resources if you create a
     stack from this template.
 Parameters:
-    ResourceTagPrefix:
-        Type: String
-        AllowedPattern: '[0-9a-zA-Z@.#-]+'
-        MaxLength: 50
-        ConstraintDescription: >-
-            Resource tag prefix can include numbers, lowercase letters,
-            uppercase letters, ampersat(@) , hyphens (-), period (.), and hash
-            (#). Max length is 50.
-        Description: >-
-            The ResourceGroup Tag Key used on all resources and as the name
-            prefix of all applicable resources. Can only contain numbers,
-            lowercase letters, uppercase letters, ampersat(@), hyphens (-),
-            period (.), and hash (#). Max length is 50.
-    CustomIdentifier:
-        Type: String
-        Default: fgtASG
-        MaxLength: '10'
-        AllowedPattern: '[A-Za-z0-9]+'
-        ConstraintDescription: must only contain uppercase and lowercase letters and numbers
-        Description: >-
-            An alternative name prefix to be used on a resource that the
-            'Resource tag prefix' cannot apply to. Can only contain numbers,
-            lowercase letters, and uppercase letters. Max length is 10.
-    S3BucketName:
-        Type: String
-        MinLength: 3
-        MaxLength: 63
-        AllowedPattern: '^[0-9a-zA-Z]+([0-9a-zA-Z-.]*[0-9a-zA-Z])*$'
-        ConstraintDescription: >-
-            Deployment resource S3 bucket name can include numbers, lowercase
-            letters, uppercase letters, period (.), and hyphens (-). It cannot
-            start or end with a hyphen (-).
-        Description: >-
-            Name of the S3 bucket that contains the FortiGate Autoscale
-            deployment package. Can only contain numbers, lowercase letters,
-            uppercase letters, period (.), and hyphens (-). It cannot start or
-            end with a hyphen (-).
-    S3KeyPrefix:
+    AutoscaleNotificationSubscriberEmail:
+        AllowedPattern: ^$|^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$
+        ConstraintDescription: must be a valid email address.
+        Description: The email address (AWS SNS Topic subscriber) to receive Autoscale notifications. If provided, the template can only accept one email address. An email will be sent to the address to confirm the subscription.
```

Comment:
```suggestion
        Description: The email address (AWS SNS topic subscriber) to receive Autoscale notifications. If provided, the template can only accept one email address. An email is sent to the address to confirm the subscription.
```

Reply:
Hi @ilee-fortinet , ***AWS SNS Topic*** is treated as a name. Can you please update your suggestion?

Reply:
Updated